.github/workflows/e2e_additional_editor_and_player.yml

```diff
@@ -42,7 +42,9 @@ jobs:
       - name: Run Additional Player E2E Test
         if: startsWith(github.head_ref, 'update-changelog-for-release') == false
         run: xvfb-run -a python -m scripts.run_e2e_tests --skip-build --skip-install --suite="additionalPlayerFeatures" --prod_env
-      - uses: actions/upload-artifact@v2
+      - name: Uploading protractor-screenshots as Artifacts
+        if: ${{ always() }}
```

Comment:
I think we still need the `startsWith(github.head_ref, 'update-changelog-for-release') == false` part. Otherwise, this will try (and fail) to upload artifacts even when we skipped all the previous tests.

Also, could you try inducing a failure to confirm that this works?